kubectl-cloudflow/cmd/configure.go

```diff
@@ -66,9 +66,11 @@ func (c *configureApplicationCMD) configureImpl(cmd *cobra.Command, args []strin
 	}
 
 	// Get the Cloudflow operator ownerReference
-	ownerReference := version.GetOwnerReferenceForCloudflowOperator()
+	ownerReference := applicationCR.GenerateOwnerReference()
+
+	streamletNameSecretMap := deploy.CreateSecretsData(&applicationCR.Spec, configurationKeyValues)
+	streamletNameSecretMap = deploy.UpdateSecretsWithOwnerReference(ownerReference, streamletNameSecretMap)
```

Comment:
If we delete the app CR, all resources are removed rigjt? or is anything that is pending to complete this fix?

Reply:
That is the intention, the secrets was previously tied to the operator, but that was more of a hack.